src/app/groups/containers/managed-group-list/managed-group-list.component.html

```diff
@@ -1,16 +1,16 @@
 <alg-error
-    *ngIf="state === 'error'; else noError"
-    class="dark"
-    icon="fa fa-exclamation-triangle"
-    i18n-message message="Error while loading the group you manage"
-    [showRefreshButton]="true"
-    (refresh)="fetchData()"
+  *ngIf="state === 'error'; else noError"
+  class="dark"
+  icon="ph-duotone ph-warning-circle"
+  i18n-message message="Error while loading the group you manage"
+  [showRefreshButton]="true"
+  (refresh)="fetchData()"
 ></alg-error>
 
 <ng-template #noError>
   <p-table
     #table
-    class="alg-table --managed-group-list"
+    class="alg-table"
     [value]="data"
     [customSort]="true"
     sortMode="multiple"
```

Comment:
Isn't it because of this that there is this "1" next to the search field when searching? If so, remove it (there is only one sortable column anyway)
![Screenshot 2023-12-19 at 09 26 12](https://github.com/France-ioi/AlgoreaFrontend/assets/1053150/fcf559c9-0607-4a9c-a3ef-9d21ce7dfa0c)

BTW, it is easy to allow sorting by "name", it could be nice (still not allowing multiple sort). If not, we can leave that for later.